sherpa/astro/__init__.py

```diff
@@ -16,5 +16,14 @@
 #  with this program; if not, write to the Free Software Foundation, Inc.,
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+'''Physical constants for astronomical spectra
 
-# Nothing in here yet
+This module contains some physical constants needed to perform conversions
+that are common in the `sherpa.astro` module, e.g. from energy to wavelength.
+'''
+
+hc = 12.39841874  # nist.gov in [keV-Angstrom]
```

Comment:
it would be good to cite the actual nist.gov (or other source) where the exact constant value used was pulled from as within nist.gov there may be various pages w/ slightly different precision values.

Reply:
I agree. That would be good. But I did not look this up in nist.gov I merely moved code that we had before to a different location and unfortunately I don't know who pulled that constant in an where he or she found it because this pre-dates the move to git: https://github.com/sherpa/sherpa/blame/572231b222136fa829590be9523da0d529328bc6/sherpa/astro/utils/__init__.py#L47

Reply:
I dug through clearcase and found that the constant was modified on 05/30/08 and changed from:

       _hc = 12.398419058  # h*c in keV-Angstrom
to:
       _hc = 12.39841874  # nist.gov in [keV-Angstrom]

release notes posted to sherpadev included the note:

     o Adjusted hc constant in sherpa/astro/data.py to nist.gov standard.

Reply:
@wmclaugh Thanks for checking. I suggest we just leave it at the current value with the current comment.